src/ci/docker/host-x86_64/mingw-check-2/Dockerfile

```diff
@@ -27,6 +27,7 @@ COPY scripts/sccache.sh /scripts/
 RUN sh /scripts/sccache.sh
 
 ENV SCRIPT \
+        python3 ../x.py check --stage 0 && \
```

Comment:
I am not entirely sure about this yet. If it makes the pipeline too long, we can consider moving it to some other PR pipeline.

Reply:
Not needed anymore.